tools/sstable_manager_service.hh

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2024-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#pragma once
+
+#include "db/large_data_handler.hh"
+#include "db/cache_tracker.hh"
+#include "db/config.hh"
+#include "gms/feature_service.hh"
+#include "sstables/sstable_directory.hh"
+#include "sstables/sstables_manager.hh"
+
+struct sstable_manager_service {
```

Comment:
I think we should extract/generalize more than just this struct. See my comment on the next commit.

Reply:
done.